mqtt/mqtt-broker/benches/file_persist_bench.rs

```diff
@@ -121,17 +121,19 @@ fn make_fake_state(
 
     let sessions = (0..num_clients)
         .map(|i| {
-            let mut session = SessionState::new(ClientId::from(format!("Session {}", i)));
             let waiting_to_be_sent = (0..num_unique_messages)
                 .map(|_| make_fake_publish(format!("Topic {}", i)))
                 .chain(shared_messages.clone());
```

Comment:
```suggestion
                .chain(shared_messages.clone()).collect();
```
It will produce VecDeque and you don't need to specify exact type below